src/NServiceBus.Core/DelayedDelivery/TimeoutManager/ExpiredTimeoutsPoller.cs

```diff
@@ -81,18 +83,13 @@ public void NewTimeoutRegistered(DateTime expiryTime)
             while (!cancellationToken.IsCancellationRequested)
             {
                 await SpinOnce().ConfigureAwait(false);
+                await Task.Delay(NextRetrievalPollSleep, cancellationToken).ConfigureAwait(false);
```

Comment:
This will make SpinOnce and as result query to be fired every second. Is that what we want to do?